cli/tools/coverage.rs

```diff
@@ -296,30 +289,46 @@ fn collect_coverages(dir: &PathBuf) -> Result<Vec<ScriptCoverage>, AnyError> {
 
 fn filter_coverages(
   coverages: Vec<ScriptCoverage>,
-  exclude: Vec<Url>,
+  include: Vec<String>,
+  exclude: Vec<String>,
 ) -> Vec<ScriptCoverage> {
+  let include_patterns = include
+    .iter()
+    .map(|i| glob::Pattern::new(i).unwrap())
+    .collect::<Vec<glob::Pattern>>();
+  let exclude_patterns = exclude
+    .iter()
+    .map(|x| glob::Pattern::new(x).unwrap())
+    .collect::<Vec<glob::Pattern>>();
+
```

Comment:
Because these come from file urls and therefore are absolute; probably conflicts with user expectations.